src/App.vue

```diff
@@ -26,11 +26,14 @@ export default {
     };
   },
   computed: {
+    headerClass() {
+      return  this.$router.currentRoute.value.name === 'Rules' ? 'sticky': '';
+    },
```

Comment:
I think it'd be preferable for this to always be sticky instead of only on this page